deployments/liqo/templates/_helpers.tpl

```diff
@@ -0,0 +1,105 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "liqo.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "liqo.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "liqo.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "liqo.labels" -}}
+helm.sh/chart: {{ include "liqo.chart" . }}
+{{ include "liqo.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "liqo.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "liqo.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
```

Comment:
I suppose this could be parametrized to take the component name (and optionally the module name) as argument, hence avoiding `advertisement.selectorLabels` and so on. Additionally, I would stick to standard k8s labels instead of custom ones (https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/):

I would propose something like:
```
app.kubernetes.io/name: <component-name> (e.g. advertisement)
app.kubernetes.io/component: <module-name> (e.g. sharing-protocol)
app.kubernetes.io/part-of: liqo
```

The definition of the common labels above should also be adapted with the parameters.